client-spark/spark3/src/main/java/org/apache/spark/shuffle/RssShuffleManager.java

```diff
@@ -261,7 +283,8 @@ public <K, V, C> ShuffleHandle registerShuffle(int shuffleId, ShuffleDependency<
         id.get(),
         dependency.rdd().getNumPartitions(),
         dependency,
-        partitionToServers);
+        partitionToServers,
+        remoteStorage);
```

Comment:
MR Appmaster also leverages `getShuffleAssignments`

Reply:
ok, I will do the update for MR

Reply:
@frankliee I prefer to have a new PR to add this feature for MR